aten/src/ATen/native/BackendSelectRegister.cpp

```diff
@@ -0,0 +1,54 @@
+#include <ATen/ATen.h>
+#include <ATen/Dispatch.h>
+#include <ATen/core/op_registration/op_registration.h>
+#include <c10/core/TensorOptions.h>
+
+// TODO:
+// 1) add all factory functions
+// 2) autogen this
+
+namespace at {
+namespace native {
+  Tensor empty_backend(IntArrayRef size, const TensorOptions & options, c10::optional<MemoryFormat> memory_format) {
```

Comment:
is there a reason these aren't in an anonymous namespace (with the "plain" name, i.e. without the backend suffix)?